cli/src/main/java/keywhiz/cli/commands/AddAction.java

```diff
@@ -166,6 +167,19 @@ private void createAndAssignSecret(String secretName, byte[] content, boolean us
     return metadata;
   }
 
+  private long getExpiry() {
+    String expiry = addActionConfig.expiry;
+    if (expiry != null && !expiry.isEmpty()) {
```

Comment:
On `expiry.isEmpty()` should probably throw an error, because it means caller passed `--expiry=''` or such which is incorrect and should fail.